Automattic/wp-calypso
# Remove Jetpack Cloud Dashboard

#### Changes proposed in this Pull Request

* Remove "Dashboard" from sidebar
* Add redirect from root route
* Add in auto-expanding sidebar sections

<img width="1035" alt="Screen Shot 2020-03-25 at 4 12 34 PM" src="https://user-images.githubusercontent.com/2810519/77594411-aedad600-6eb3-11ea-842f-f21519c63551.png">


#### Testing instructions

1. Navigate to `/` with all features on. Verify you pick a site then are taken to `/backups`
2. Reload the browser and verify that the "backups" section of the sidebar stays expanded
3. Disable the "jetpack-cloud/backups" feature ( set "jetpack-cloud/backups" to `false` in config/jetpack-cloud-development.json and _restart calypso_ )
4. Repeat steps 1 & 2, verifying you were sent to `/scan` instead
